.travis.yml

```diff
@@ -66,6 +66,19 @@ matrix:
         - DESC="package no error"
         - CMD="mvn clean package -Passembly"
 
+    # checkstyle regression no exception (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle regression no exception"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cd checkstyle &&"
+        - CMD3="git checkout -b test-branch &&"
+        - CMD4="git checkout master &&"
+        - CMD5="git reset --hard HEAD^ &&"
+        - CMD6="cd .. &&"
+        - CMD7="mvn clean package -Passembly"
+        - CMD8="java -jar target/regression-tool-1.0-SNAPSHOT-all.jar -r checkstyle -p test-branch --stopAfterGenerateConfig"
+
```

Comment:
Travis shows no result.
You built all CMDXs but you didn't combine them into `CMD` (no numbers).